src/terminal.rs

```diff
@@ -0,0 +1,50 @@
+use crate::Todo;
+use console::Term;
+use std::io::Result;
+
+pub struct Terminal {
+    term: Term,
+}
+impl Terminal {
+    pub fn new() -> Self {
+        Terminal {
```

Comment:
Bobeirinha, mas fica mais fácil renomear a struct caso precisarmos:

```suggestion
        Self {
```